_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Fixed a bug that led to the `querySummary` field in the [`crdb_internal.statements_statistics`](../{{site.versions["dev"]}}/crdb-internal.html) metadata column being empty. [#84170][#84170]
+- Previously, CockroachDB could deadlock when evaluating analytical queries if multiple queries had to spill to disk at the same time. This is now fixed by making some of the queries error out instead. [#84398][#84398]
+- Fixed a bug where an ephemeral I/O error could crash a node. [#84449][#84449]
+- This PR fixed a bug where, in a [`ALTER PRIMARY KEY`](../{{site.versions["dev"]}}/alter-primary-key.html), if the new primary key columns is a subset of the previous primary key columns, CockroachDB will not rewrite existing secondary indexes, and hence those secondary indexes continue to have some of the previous primary key columns in their `suffixColumns`. But the user might, reasonably think those columns are not used anymore and proceed to drop them. The bug then caused those dependent secondary indexes to be dropped, unexpectedly for the user. [#84303][#84303]
+- [CLI `cockroach` commands](../{{site.versions["dev"]}}/cockroach-commands.html) connecting to a remote server will not produce spurious `"latency jump"` warnings any more. This bug had been introduced in CockroachDB v21.2. [#84031][#84031]
+- This patch fixes vectorized evaluation of `COALESCE` involving expressions of type `VOID` and enhances type checking of `NULLIF` expressions with `VOID`, so incompatible comparisons can be caught during query compilation instead of during query execution. [#83868][#83868]
+- Changing the time window using arrow buttons and the **Now** button will now properly turn the timeframe into a moving window when `endTime = now`. [#84649][#84649]
+- The server process no longer announces that it is shutting down on `stdout` when running with `--background`. [#84532][#84532]
+- The `public` role can no longer be granted default privileges with the `grant` option. This was a bug because the `public` role already cannot have the grant option on regular privileges. [#85027][#85027]
+- Fixed a bug that where CockroachDB should initialize a `schemaChangerState` of `connExecutor` from the corresponding session variable (`use_declarative_schema_changer`), which can cause DDL statements to be executed under the legacy schema changer unknowingly. [#85344][#85344]
+- When a CockroachDB [node is being drained](../{{site.versions["dev"]}}/node-shutdown.html#drain-a-node-manually), all queries that are still running on that node are now forcefully canceled after waiting the `server.shutdown.query_wait` period. [#82752][#82752]
+- The SQL unix socket, when requested, now contains a port number compatible with the connection URL when `--listen-addr` is configured to auto-allocate a port number. This bug had existed since CockroachDB v1.0. [#84910][#84910]
```

Comment:
We can link: `[Unix socket](../{{site.versions["dev"]}}/connection-parameters.html#example-uri-for-a-unix-domain-socket`